src/test/ui/const-generics/invalid-enum.stderr

```diff
@@ -30,44 +30,24 @@ error[E0747]: type provided when a constant was expected
    |
 LL |   let _: Example<CompileFlag::A, _> = Example { x: 0 };
    |                  ^^^^^^^^^^^^^^
-   |
-help: if this generic argument was intended as a const parameter, try surrounding it with braces:
-   |
-LL |   let _: Example<{ CompileFlag::A }, _> = Example { x: 0 };
-   |                  ^                ^
```

Comment:
These are no longer shown because I think they resolve to error, we'd probably want to keep these in but just left it here as an option in case.

Reply:
Yeah, we still want these suggestions. What happens if you remove the `Err` match arm?

Reply:
W/o the err match arm, these should come back